dashboard/pkg/epinio/pages/c/_cluster/dashboard.vue

```diff
@@ -165,6 +208,24 @@ export default Vue.extend<any, any, any, any>({
 
 <template>
   <div class="dashboard">
+    <Banner
```

Comment:
Banner should go below the welcome card. the vertical spacing above and below should match the existing spacing between welcome card and namespace/app/service cards